advanced_function/task1.js

```diff
@@ -0,0 +1,17 @@
+function handleNum(num, isEven,isOdd){
```

Comment:
isEven,isOdd - I would suppose it should be boolean values

use more informative names like `onEven,onOdd ` or `handleEven,handleOdd `